<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeScreen</class>
 <widget class="QMainWindow" name="HomeScreen">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Real Time Eye Checkup</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>541</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>assets/img/icon.png</pixmap>
    </property>
   </widget>
<!--   <widget class="QLineEdit" name="username">-->
<!--    <property name="geometry">-->
<!--     <rect>-->
<!--      <x>320</x>-->
<!--      <y>380</y>-->
<!--      <width>261</width>-->
<!--      <height>41</height>-->
<!--     </rect>-->
<!--    </property>-->
<!--    <property name="font">-->
<!--     <font>-->
<!--      <family>MS Sans Serif</family>-->
<!--      <pointsize>14</pointsize>-->
<!--     </font>-->
<!--    </property>-->
<!--    <property name="text">-->
<!--     <string/>-->
<!--    </property>-->
<!--    <property name="echoMode">-->
<!--     <enum>QLineEdit::Normal</enum>-->
<!--    </property>-->
<!--   </widget>-->
<!--   <widget class="QLabel" name="label_2">-->
<!--    <property name="geometry">-->
<!--     <rect>-->
<!--      <x>190</x>-->
<!--      <y>380</y>-->
<!--      <width>121</width>-->
<!--      <height>41</height>-->
<!--     </rect>-->
<!--    </property>-->
<!--    <property name="font">-->
<!--     <font>-->
<!--      <family>MS Sans Serif</family>-->
<!--      <pointsize>14</pointsize>-->
<!--     </font>-->
<!--    </property>-->
<!--    <property name="text">-->
<!--     <string>Name:</string>-->
<!--    </property>-->
<!--    <property name="alignment">-->
<!--     <set>Qt::AlignCenter</set>-->
<!--    </property>-->
<!--   </widget>-->
<!--   <widget class="QLabel" name="label_3">-->
<!--    <property name="geometry">-->
<!--     <rect>-->
<!--      <x>190</x>-->
<!--      <y>430</y>-->
<!--      <width>121</width>-->
<!--      <height>41</height>-->
<!--     </rect>-->
<!--    </property>-->
<!--    <property name="font">-->
<!--     <font>-->
<!--      <family>MS Sans Serif</family>-->
<!--      <pointsize>14</pointsize>-->
<!--     </font>-->
<!--    </property>-->
<!--    <property name="text">-->
<!--     <string>City:</string>-->
<!--    </property>-->
<!--    <property name="alignment">-->
<!--     <set>Qt::AlignCenter</set>-->
<!--    </property>-->
<!--    <property name="wordWrap">-->
<!--     <bool>false</bool>-->
<!--    </property>-->
<!--   </widget>-->
<!--   <widget class="QLineEdit" name="password">-->
<!--    <property name="geometry">y-->
<!--     <rect>-->
<!--      <x>320</x>-->
<!--      <y>430</y>-->
<!--      <width>261</width>-->
<!--      <height>41</height>-->
<!--     </rect>-->
<!--    </property>-->
<!--    <property name="font">-->
<!--     <font>-->
<!--      <family>MS Sans Serif</family>-->
<!--      <pointsize>14</pointsize>-->
<!--     </font>-->
<!--    </property>-->
<!--    <property name="inputMethodHints">-->
<!--     <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>-->
<!--    </property>-->
<!--    <property name="echoMode">-->
<!--     <enum>QLineEdit::Password</enum>-->
<!--    </property>-->
<!--    <property name="alignment">-->
<!--     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>-->
<!--    </property>-->
<!--    <property name="placeholderText">-->
<!--     <string notr="true"/>-->
<!--    </property>-->
<!--   </widget>-->
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>490</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Sans Serif</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Get Started</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Papyrus</family>
      <pointsize>15</pointsize>
      <weight>40</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="text">
     <string>Real Time Eye Checkup</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
